README.md

```diff
@@ -45,4 +56,10 @@ $ pip install karton-archive-extractor
 $ karton-archive-extractor
 ```
 
+## Running in Docker
+
+Sflock uses [ZipJail](https://github.com/hatching/tracy/tree/master/src/zipjail) as a usermode syscall filtering mechanism. As a result, the karton service has to be able to use `ptrace` so remember to add the `SYS_PTRACE` capability if using Docker.
```

Comment:
```suggestion
Sflock uses [ZipJail](https://github.com/hatching/tracy/tree/master/src/zipjail) as a usermode syscall filtering mechanism. As a result, the karton service has to be able to use `ptrace` so ensure that your container has `SYS_PTRACE` capability.
```

Maybe add example docker invocation :thinking: 

Also I don't think this is true - `SYS_PTRACE` is not necessary to trace your own children. Otherwise it would be impossible to use gdb without root, for example (non-root processes usually have no capabilities at all). Problem is somewhere else, probably :thinking:. Maybe rephrase that as a `debugging step`?